chapters/chapter1.1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Chapter 1.1: What is ML?'
+description:
+  ' This Chapter will explain what ML is.'
+prev: null
+next: /chapter1.2
+type: chapter
+id: 1
+---
+
+<exercise id="1" title="Video Lecture">
+
+<iframe width="100%" height="480" src="https://www.youtube.com/embed/CCzx4UDkzpA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+</exercise>
+
+<exercise id="2" title="Slides">
+
+<object data="pdfs/slides-basics-whatisml.pdf
+" type="application/pdf" style="width:100%;height:480px">
+    <embed src="pdfs/slides-basics-whatisml.pdf
+" type="application/pdf" />
+</object>
+
+</exercise>
+
+<exercise id="3" title="Demo">
+
+<iframe width="100%" height="480" src="https://jjallaire.github.io/deep-learning-with-r-notebooks/notebooks/2.1-a-first-look-at-a-neural-network.nb.html"></iframe>
+
+</exercise>
+
+<exercise id="4" title="Quiz">
```

Comment:
Seems like I get a "that's correct" if I check *any* of the correct answers. What we want would be that I only get it if I check *all* of the correct answers. Can you find out if that is possible?

Reply:
I just search on the internet, and I did not find any solution for that. Do you have any idea how I can resolve this issue? In particular, can you please inform me of the GitHub source file for the quiz in the following link?

https://compstat-lmu.shinyapps.io/tutorial-ml-basics/#section-data-and-tasks

Reply:
Ah yes, I should have provided this in the beginning: https://github.com/compstat-lmu/lecture_i2ml_learnr_tutorials

Not sure if this helps with the issue, but let's see ;)

Reply:
Dear Heidi, during the last days, I implemented several solutions to fix this issue, but I could not resolve it. Although my methods were unsuccessful, I think you would be better to know them; perhaps another person can utilize each of these candidate options and resolve the issue.

**1. The first option** is to find of way of adding a code chunk (containing R codes) to our .md files. The following link provides an example of this option:

https://rstudio.github.io/learnr/questions.html


**2. The second option** is to like what the following link suggests. To use this option, one has to define <choiceresponse> and <checkboxgroup>.

https://raw.githubusercontent.com/edx/edx-platform/0438878ab96e8be21c4f75fa3979c4eb5b491543/common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

**3. The third possibility** is to re-define the structure of our quizzes in a way that each quiz has only one correct choice. While I am not sure whether we can call this as a "real" option, I noticed that they had used this option in the following course:

https://course.spacy.io/en/

https://raw.githubusercontent.com/ines/spacy-course/master/chapters/en/chapter3.md 